<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoInputBar1080p</class>
 <widget class="QDialog" name="VideoInputBar1080p">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1339</width>
    <height>129</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>4</y>
     <width>1331</width>
     <height>121</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="buttonChAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh3">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh5">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh6">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh7">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonCh8">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMirror">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonFlip">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonClose">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>10</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
